Modules/Invoices/Services/InvoiceGroupService.php

```
@@ -31,6 +37,16 @@ public function generateInvoiceNumber(InvoiceGroup $invoiceGroup, bool $setNext
         return $identifier;
     }
 
+    /**
+     * Get all invoice groups.
+     *
+     * @return \Illuminate\Database\Eloquent\Collection
+     */
+    public function getAll()
+    {
+        return InvoiceGroup::all();
```

PHPMD (2.15.0)

Avoid using static access to class '\Modules\Invoices\Models\InvoiceGroup' in method 'getAll'. (undefined)

(StaticAccess)